clang/include/clang/CIR/Dialect/IR/CIRAttrs.td

```diff
@@ -92,6 +92,20 @@ def CIR_BoolAttr : CIR_Attr<"Bool", "bool", [TypedAttrInterface]> {
   }];
 }
 
+//===----------------------------------------------------------------------===//
+// UninitializedUnionFieldAttr
+//===----------------------------------------------------------------------===//
+
+def UninitializedUnionFieldAttr : CIR_Attr<"UninitializedUnionField", "uninitialized_union_field", [TypedAttrInterface]> {
```

Comment:
80-cols :)

Reply:
While here, a more terse name suggestion: `InactiveUnionFieldAttr` and `inactive_field`. We also need a verifier for this attribute to make sure it's only used on const_struct unions.

Reply:
Done with renaming. But maybe I need suggestion to verify how an attribute is used. I take a look at `CIRAttrs.cpp` and `mlir/include/mlir/IR/Attributes.h` but failed to find such interface.

Reply:
`mlir::cir::ConstArrayAttr::verify` is one example. Verifier can be enabled with `let genVerifyDecl = 1;`

Reply:
It looks like `mlir::cir::ConstArrayAttr::verify` only checks itself too. But what we want here is to check the parent or the context of the attribute. It is pretty different.

Reply:
Good point, we cannot check the context of an attribute, as opposed to operations!